server/controllers/lecture-controller.js

```diff
@@ -343,7 +344,7 @@ const sendEmailsForCancelledLecture = async (lectureId) => {
       { name: "user.name" },
       { surname: "user.surname" },
       { email: "user.email" },
-      { lectureName: "lecture.name" },
+      //{ lectureName: "lecture.name" },
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.